corehq/messaging/scheduling/forms.py

```diff
@@ -2873,6 +2877,10 @@ def add_additional_content_types(self):
             self.fields['content'].choices += [
                 (self.CONTENT_CUSTOM_SMS, _("Custom SMS")),
             ]
+        if HQ_FCM_UTIL:
```

Comment:
I guess using the `is_fcm_available` everywhere function would bring consistency. We could also memoize it.

Reply:
Git rid of the constant and now using settings.FCM_CREDS.
9c1da061e19ee0ba06d639b7e8a2b1b7f5232180